modules/contracts/apps/etf/src/state.rs

```diff
@@ -0,0 +1,14 @@
+use abstract_core::objects::fee::Fee;
+use cosmwasm_std::Addr;
+use cw_storage_plus::Item;
+
+/// State stores LP token address
+/// BaseState is initialized in contract
+#[cosmwasm_schema::cw_serde]
+pub struct State {
+    pub share_token_address: Addr,
+    pub manager_addr: Addr,
+}
+
+pub const STATE: Item<State> = Item::new("\u{0}{5}state");
+pub const FEE: Item<Fee> = Item::new("\u{0}{3}fee");
```

Comment:
Would be nice to add comment why we start state key with NULL

Reply:
Is it common practice to do so? I haven't seen that elsewhere 👀

Reply:
> Is it common practice to do so? I haven't seen that elsewhere eyes

Same, haha. Saw just random bytes before(to decrease key length), but not a normal key starting with NULL

Reply:
Ah, yeah that used to be required in CW denoting how long the key was. state = 5 chars, fee = 3 chars.

Reply:
Oh wow, how old is this contract

Reply:
> Oh wow, how old is this contract

Howard wrote it as one of the first Abstract modules >1 year ago.